### PR TITLE
feat(index): Fix #193 add availableLocales

### LINKED ISSUE #193: List of available languages
I'd like to have an feature which allows me to output all available languages with one method. That would allow to make a list of all available translations.

### PR DESCRIPTION
**Pull request by exoego**
Closes #193

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -169,6 +169,7 @@ export default class VueI18n {
   get messages (): LocaleMessages { return looseClone(this._getMessages()) }
   get dateTimeFormats (): DateTimeFormats { return looseClone(this._getDateTimeFormats()) }
   get numberFormats (): NumberFormats { return looseClone(this._getNumberFormats()) }
+  get availableLocales (): string[] { return Object.keys(this.messages) }
```
Comment (kazupon) [inline src/index.js]:
```suggestion
  get availableLocales (): Locale[] { return Object.keys(this.messages) }
```

**Comment by kazupon** [comment]
and,  in addition, I have one more request.
Could you add type declaration of typescript and flow, please? :pray:

**Comment by exoego** [comment]
@kazupon 
Docs and type definition added !!

File: decls/i18n.js [hunk 2]
```diff
@@ -85,6 +85,8 @@ declare interface I18n {
   set fallbackLocale (locale: Locale): void,
   get messages (): LocaleMessages,
   get dateTimeFormats (): DateTimeFormats,
+  get numberFormats (): NumberFormats,
```
Comment (exoego) [inline decls/i18n.js]: Note) 
I thinks `numberFormats` is missing so I fixed, though it is out of scope of #193...
I can make a separate PR if it is preferrable.

Comment (kazupon) [reply decls/i18n.js]: I hope you will separate it as PR. 🙏

Comment (exoego) [reply decls/i18n.js]: Fixed!

File: vuepress/api/README.md [hunk 3]
```diff
@@ -209,6 +209,19 @@ The number formats of localization.
 
   * **See also:** `NumberFormats` type of [flowtype definitions](https://github.com/kazupon/vue-i18n/blob/dev/decls/i18n.js)
 
+#### availableLocales
+
+> :new: 8.8.3+
```
Comment (kazupon) [inline vuepress/api/README.md]:
```suggestion
> :new: 8.9.0+
```

**Comment by kazupon** [comment]
CI still have been failing.
Could you fix it, please?

**Comment by rimiti** [comment]
Good job 🚀

**Pull request merged**